adapters/appnexus/appnexus.go

```diff
@@ -260,17 +263,11 @@ func preprocess(imp *openrtb2.Imp, appnexusExt *openrtb_ext.ExtImpAppnexus, defa
 	}
 
 	// Populate imp.displaymanagerver if the SDK failed to do it.
-	if len(imp.DisplayManagerVer) == 0 && len(defaultDisplayManagerVer) > 0 {
-		imp.DisplayManagerVer = defaultDisplayManagerVer
+	if len(imp.DisplayManagerVer) == 0 && len(displayManagerVer) > 0 {
+		imp.DisplayManagerVer = displayManagerVer
 	}
 
-	impExt := appnexusImpExt{Appnexus: appnexusImpExtAppnexus{
-		PlacementID:       int(appnexusExt.PlacementId),
-		TrafficSourceCode: appnexusExt.TrafficSourceCode,
-		Keywords:          makeKeywordStr(appnexusExt.Keywords),
-		UsePmtRule:        appnexusExt.UsePaymentRule,
-		PrivateSizes:      appnexusExt.PrivateSizes,
-	}}
+	impExt := buildImpExt(appnexusExt)
```

Comment:
Here `buildImpExt` is only initialises `appnexusImpExt`. This can happen in `buildRequestImp`. No need for separate function